gulpfile.js

```diff
@@ -0,0 +1,61 @@
+var gulp = require('gulp'),
+    sass = require('gulp-sass'),
+    concat = require('gulp-concat'),
+    minifyCSS = require('gulp-minify-css'),
+    browserSync = require('browser-sync').create()
+    ;
+
+gulp.task('default', ['css', 'js', 'fonts']);
+
+gulp.task('css', function () {
+    gulp.src(['src/scss/app.scss'])
```

Comment:
@DavidBadura this gulp stream should be returned, so gulp is knowing then this task is finished